plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -409,6 +410,9 @@ public function log_errors() {
 			$mc_stats->add( 'error', 'fatal-errors-during-shutdown' );
 			$mc_stats->do_server_side_stats();
 
+			$container->get( RemoteLogger::class );
+			$remote_logger->handle( time(), WC_Log_Levels::CRITICAL, $message, $context );
```

Comment:
This will fail ... we probably want to do:

```suggestion
			$remote_logger = $container->get( RemoteLogger::class );
			$remote_logger->handle( time(), WC_Log_Levels::CRITICAL, $message, $context );
```

Or possibly:

```suggestion
			$container->get( RemoteLogger::class )->handle( time(), WC_Log_Levels::CRITICAL, $message, $context );
```